Source/WebCore/platform/graphics/ComplexTextController.cpp

```diff
@@ -323,10 +323,10 @@ void ComplexTextController::collectComplexTextRuns()
     unsigned currentIndex = 0;
     unsigned indexOfFontTransition = 0;
 
-    const Font* font = nullptr;
-    const Font* nextFont = nullptr;
-    const Font* synthesizedFont = nullptr;
-    const Font* smallSynthesizedFont = nullptr;
+    RefPtr<const Font> font = nullptr;
+    RefPtr<const Font> nextFont = nullptr;
+    RefPtr<const Font> synthesizedFont = nullptr;
+    RefPtr<const Font> smallSynthesizedFont = nullptr;
```

Comment:
```suggestion
    RefPtr<const Font> font;
    RefPtr<const Font> nextFont;
    RefPtr<const Font> synthesizedFont;
    RefPtr<const Font> smallSynthesizedFont;
```

Nit - the `= nullptr;` are no longer necessary since the default ctor will initialize to `nullptr`.